docs/gitbook/SUMMARY.md

```diff
@@ -25,6 +25,7 @@
     * [Installation](getting_started/installation.md)
     * [Install as permanent functions](getting_started/permanent-functions.md)
     * [Input Format](getting_started/input-format.md)
+    * [Step-by-Step Tutorial on Supervised Learning](getting_started/tutorial.md)
```

Comment:
Better moved to `Supervised Learning` or `Regression` section or  with renaming.

Reply:
Will move under `Supervised Learning`.